includes/discovery/sensors/runtime/apc.inc.php

```diff
@@ -15,3 +15,65 @@
     $high_limit         = 3000;
     discover_sensor($valid['sensor'], 'runtime', $device, $oid, $index, $type, $descr, $divisor, '1', $low_limit, $low_limit_warn, $warn_limit, $high_limit, $current);
 }
+
+// InRow IRRP100
+$oids = snmp_get($device, 'airIRRP100GroupSetpointsCoolMetric.0', '-OsqnU', 'PowerNet-MIB');
```

Comment:
On the APC Air Conditionner, you can set an alert (number of weeks of runtime) on a few parts runtime (air filter, comrpessor,...). This alert can be enabled or not.
With the dedicated apc.inc.php runtime file, I just want to check if these alerts are enabled or not. If they are, we need to set the $high_limit value with the value (number of weeks converted in minutes) set in the Air Conditionner. If the alarm is disabled, the $high_limit is set to null.
I don't know if we can do this in the yaml file...